numpy/random/mtrand/mtrand.pyx

```diff
@@ -4549,7 +4549,7 @@ cdef class RandomState:
 
         Parameters
         ----------
-        n : int
+        n : int or array_like of ints
```

Comment:
Needs `.. versionchanged:: 1.14.0` with a note that allowing array_like here is new. You can find examples in the file.